### ISSUE TITLE
VIP Enterprise Search breaks byline lookup via API

### ISSUE THREAD

**Issue by scottnelle** [alleyinteractive/byline-manager#120]
### Description of the bug

When WordPress.com VIP Enterprise Search query offloading is enabled, it breaks byline lookup unless byline profiles are indexed.

### Steps To Reproduce

On a site Enterprise Search query offloading enabled and with byline profiles excluded from the elasticsearch index, search for an author profile on the post edit screen, either in the classic editor or the block editor. The search API will always return an empty array because there are no profiles to return in the index.

**Comment by renatonascalves** [comment]
This is a known issue and not technically a Byline Manager problem, even though the plugin could be part of the solution. 

But a consideration here is that it is up to the site owner to decide if profiles are indexed or not.

**Comment by scottnelle** [comment]
Yeah, that makes sense. Maybe we could add a note in the documentation. Out of the box I think it works fine with Enterprise search, but it seems pretty counter-intuitive that removing bylines from the front-end search breaks the plugin's back end functionality.

**Comment by renatonascalves** [comment]
> it seems pretty counter-intuitive that removing bylines from the front-end search breaks the plugin's back end functionality.

By that I mean you are removing the post type from being indexed. One can remove the BM post type from the list of accepted post types in a search on the front-end. And the back-end/editor would continue to work.

**Comment by scottnelle** [comment]
I get that. I should have been more clear. The thing that feels counter-intuitive to me is that the search endpoint behaves like a front-end search. I understand from a software perspective why it happens and I was able to fix it immediately in my current project, but if I didn't know how the plugin works and also how Enterprise Search works, I would have been pretty confused. If we don't want to do anything about this that's fine, but I think a lot of developers are going to run into this issue and be less equipped to solve it for themselves, and finding some way to document it (or bypassing query offloading for this endpoint by default) might be helpful.

**Comment by renatonascalves** [comment]
I agree! I was confused myself when that happened. I'd suggest improving the documentation.
